src/test/skript/tests/syntaxes/expressions/ExprSortedList.sk

```diff
@@ -0,0 +1,18 @@
+test "sort":
+	# Populate list
+	set {_i} to 0
+	loop 5 times:
+		set {_i} to {_i} + 1
+		set {_list::%{_i}%} to a random integer from 1 to 100
+	loop 5 times:
+		set {_i} to {_i} + 1
+		set {_list::%{_i}%} to a random number from 1 to 100
+
+	# Test sorting
+	loop sorted {_list::*}:
+		if {_prev} is not set:
+			set {_prev} to loop-value
+			continue
+		assert loop-value >= {_prev} with "Couldn't sort correctly"
```

Comment:
Currently this will only set `_prev` to the very first value of the list, so will just check each value against the very first one; `(0, 3, 2, 1, 4)` could pass since all of `3, 2, 1, 4` will be greater than `_prev = 0`.

```suggestion
		if {_prev} is set:
			assert loop-value >= {_prev} with "Couldn't sort correctly"
		set {_prev} to loop-value
```
You probably want to set `_prev` at the end of each run.

Reply:
Thanks for the catch